spec/requests/cloud_networks_spec.rb

```diff
@@ -97,4 +97,19 @@
       expect(response).to have_http_status(:ok)
     end
   end
+
+  describe 'OPTIONS /api/cloud_networks' do
+    it 'returns a DDF schema for add when available via OPTIONS' do
+      zone = FactoryBot.create(:zone, :name => "api_zone")
+      provider = FactoryBot.create(:ems_network, :zone => zone)
+
+      allow(provider.class::CloudNetwork).to receive(:params_for_create).and_return('foo')
+
+      options(api_cloud_networks_url, :params => {:ems_id => provider.id})
+      options("#{api_cloud_networks_url}?ems_id=#{provider.id}")
+
+      expect(response.parsed_body['data']['form_schema']).to eq('foo')
+      expect(response).to have_http_status(:ok)
+    end
+  end
```

Comment:
Should also add an `'OPTIONS /api/cloud_networks/:id` spec here

Reply:
@agrare for this test case I currently have:
`
allow(provider.class::CloudNetwork).to receive(:params_for_edit).and_return('foo')

options(api_cloud_networks_url, :params => {})
options("#{api_cloud_networks_url}/#{provider.id}")

expect(response.parsed_body['data']['form_schema']).to eq('foo')
expect(response).to have_http_status(:ok)
`
Not sure if I am setting this up wrong but is there any way I can get the cloud_network_id from provider because the way I have it now the test case is failing due to the options call not returning anything.